v4/platformUI/src/main/java/exchange/dydx/platformui/components/container/PlatformInfoContainer.kt

```diff
@@ -110,22 +143,49 @@ data class Toast(
     val type: Type = Type.Info,
     val duration: Duration,
     val buttonAction: (() -> Unit)? = null,
+    val cancelAction: (() -> Unit)? = null,
 ) {
     enum class Type {
-        Error, Info, Warning;
+        Error, Info, Warning, Success;
 
         val backgroundColor: Color
             get() = when (this) {
                 Error -> ThemeColor.SemanticColor.color_red.color
                 Info -> ThemeColor.SemanticColor.layer_5.color
                 Warning -> ThemeColor.SemanticColor.color_yellow.color
+                Success -> ThemeColor.SemanticColor.color_purple.color
```

Comment:
Dan actually recommended deprecating the purple colors, we should use `Info` instead.

Reply:
Removed